.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -6,22 +6,23 @@ body:
     attributes:
       value: |
         Please note the following:
-        1. **Please update your Minetest Engine to the latest stable or dev version** before submitting bug reports. Make sure the bug is still reproducible on the latest version.
+        1. **Please update Luanti to the latest stable or dev version** before submitting bug reports. Make sure the bug is still reproducible on the latest version.
         2. This page is for reporting the bugs of **the engine itself**. For bugs in a particular game, please [search for the game in the ContentDB](https://content.minetest.net/packages/?type=game) and submit a bug report in their issue trackers.
-            * For example, you can submit issues about the Minetest Game (the official game of Minetest) [in its own repository](https://github.com/minetest/minetest_game/issues).
+            * For example, you can submit issues about the Minetest Game (the official game of Luanti) [in its own repository](https://github.com/minetest/minetest_game/issues).
```

Comment:
You don't have to fix this, but: Should probably not call it "the official game" anymore :thinking: